src/library/Connect/LocalEnvironmentManager.cs

```diff
@@ -415,7 +415,7 @@ private void _ReportProgress(EventLevel eventLevel, string message, params objec
         /// <summary>
         /// Re-create the Kubernetes service related environment variables.
         /// </summary>
-        private IDictionary<string, string> _CreateEnvVariablesForK8s(WorkloadInfo workloadInfo)
+        public IDictionary<string, string> CreateEnvVariablesForK8s(WorkloadInfo workloadInfo)

```

Comment:
Changing this method to public so I could add UT coverage for it. Feel free to point out if there is better way to achieve this.